nx-X11/programs/Xserver/Imakefile

```diff
@@ -577,18 +546,11 @@ $(NXWINOBJS) $(NXWINLIB) $(NXWINLIBS) $(NXWINSYSLIBS) $(NXWINNXLIBS):: $(NXWINDI
 
 #ifdef mingwArchitecture
 NXWINSERVERNAME=NXming    
```

Comment:
>> Any reason for leaving some of the Windows stuff in, if you remove other?
>
> Yeah, let's undo that for now.

Actually... Let's leave it as it is, please. I seriously thinking about removing all non-GL windows stuff, as well.